odometry_child_frame_id : "small_excavator_1_base_footprint"
odometry_frame_id : "small_excavator_1_odom"
robot_name : "small_excavator_1"
